index.js

```diff
@@ -41,6 +41,7 @@ function api_request(url, method, body, uuid, callback) {
     method: method,
     json: true,
     body: body,
+    encoding: null,
```

Comment:
@mhorne89 Perhaps would be a good idea to implement a way to customize these options.

Reply:
Agreed, for now I will test this the way it is